sdnfv/openNetVM
# Skeleton NF Functionality Improvements

## Summary:
Changes Resolved From #312 

Additional Functionality:
- Additional Command-Line Arguments
- Timer-based or Packet-based print delays

**Usage:**


| This PR includes         |          |
| ------------------------ | -------- |
| Resolves issues          |  
| Breaking API changes     |
| Internal API changes     |
| Usability improvements   | 
| Bug fixes                |
| New functionality        | X
| New NF/onvm_mgr args     | 
| Changes to starting NFs  |  X
| Dependency updates       | 
| Web stats updates        | 



## Merging notes:
 - Dependencies: None  

**TODO before merging :**



## Test Plan:
- Run NF on the manager
- Use speed_tester to send packets to skeleton
- Ensure that the skeleton NF can correctly print the number of packets. Test timer-based (-p) delays and packet-based delays (-v).


## Review: 
Review comments to confirm validity
Command-line arguments should be mutually exclusive
